types/meteor/mongo.d.ts

```diff
@@ -243,3 +267,15 @@ declare module "meteor/mongo" {
         }
     }
 }
+
+declare module MongoInternals {
+    interface Connection {
```

Comment:
To match [the source](https://github.com/meteor/meteor/blob/46e00a875726dff078c59eaf7fdc7b956cf0a996/packages/mongo/mongo_driver.js#L136):
```suggestion
    interface MongoConnection {
```